src/R/Components/Test/Microsoft.R.Components.Test.csproj

```diff
@@ -190,6 +182,14 @@
       <Project>{1f89902f-853e-44b5-93a7-e237984c4fb7}</Project>
       <Name>Microsoft.Windows.Core.Test</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\Windows\Host\Client\Impl\Microsoft.R.Host.Client.Windows.csproj">
```

Comment:
What does Components need from the Windows part? I am splitting Components next.